app/views/core/RootView.coffee

```diff
@@ -124,6 +124,9 @@ module.exports = class RootView extends CocoView
 
     if title = @getTitle() then title += ' | CodeCombat'
     else title = 'CodeCombat - Learn how to code by playing a game'
+    
+    if localStorage?.showViewNames
+      title = @constructor.name
```

Comment:
I recommend adding a `showViewNames` function to `application` so it's explicit and there's a way to autocomplete it.